packages/modular-scripts/src/build/index.ts

```diff
@@ -230,8 +230,12 @@ async function buildStandalone(
       modular: targetPackageJson.modular,
       dependencies: targetPackageJson.dependencies,
       bundledDependencies: targetPackageJson.bundledDependencies,
-      module: jsEntryPoint,
-      style: cssEntryPoint,
+      module: jsEntryPoint
+        ? path.join(paths.publicUrlOrPath, jsEntryPoint)
```

Comment:
I recommend path.join but it actually doesn't resolve relative paths... I dug a little deeper and you don't need to do path.join it seems because `path.publicUrlOrPath` has a sanitized version of `PUBLIC_URL` so if you concat them together it should be fine. 

I still think you should keep ternary so that it's set to `undefined` if no jsEntrypoint is available.

Reply:
Done + added a test so we're sure the corner case doesn't generate double slashes